lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -40,11 +40,26 @@ public function index($parameters) {
   public function show($id) {
     try {
       $reportbackItem = ReportbackItem::get($id);
-      $reportbackItem = $this->removeUnauthorizedResults($reportbackItem);
-      $reportbackItem = services_resource_build_index_list($reportbackItem, 'reportback-items', 'id');
+
+      if (!user_access('administer modules')) {
+        $reportbackItem = $this->removeUnauthorizedResults($reportbackItem);
```

Comment:
`removeUnauthorizedResults()` this method already has a check to see if `user_access('view any reportback')`... so  don't think we need another. If it turns out we actually want the rule for `administer modules` vs `view any reportback` then maybe change the conditional in `removeUnauthorizedResults()`?